File: solutions/0026-remove-duplicates-from-sorted-array/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size()<2){
            return 1;
        }
     int i = 0;
     int j  = 1;
     while(j<nums.size()){
        if(nums[i] == nums[j]){
            j++;
        }else{
            i++;
            int temp = nums[i];
            nums[i] = nums[j];
            nums[j] = temp;
            j++;
        }
     }
     return i+1;
    }
};

File: solutions/0026-remove-duplicates-from-sorted-array/solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 0;
        int n = nums.length;
        for(int i = 1;i<n;i++){
            if(nums[i] != nums[k]){
                k = k+1;
                nums[k] = nums[i];
            }
        }
        return k+1;
    }
}
